toontown/leveleditor/AutoSaver.py

```diff
@@ -26,11 +30,12 @@ def autoSaverProcess():
             while AutoSaver.autoSaverToggled is True:
                 # outputFile filename is empty, which may occur if filename is left blank in file prompt
                 if DNASerializer.outputFile is None:
-                    print('No file loaded, exiting auto saving loop')
+                    print('No file loaded, exiting auto saving loop...')
```

Comment:
If you don't mind creating a notify category and using that instead of print, that would be nice, but i'll still merge if you dont do that and just do it myself